airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/schemas/survey_questions.json

```diff
@@ -24,13 +24,52 @@
       "type": ["object", "null"]
     },
     "sorting": {
-      "type": ["string", "null"]
+      "type": ["object", "null"],
+      "properties": {
+        "ignore_last": {
+          "type": "boolean"
+        },
+        "type": {
+          "type": "string"
+        }
+      }
     },
     "required": {
-      "type": ["string", "null"]
+      "type": ["object", "null"],
+      "properties": {
+        "amount": {
+          "type": "string"
```

Comment:
we usually do all types safe with nulls (not only objects)
```
"type": "string" -> "type": ["null", "string"]
```

For example you can look on your data object which you provided:

```
ant$ cat _AIRBYTE_RAW_SURVEY_QUESTIONS.json | jq .required
{
  "amount": null,
  "text": "This question requires an answer.",
  "type": null
}
```

as you can see `amount` field can be `null` (not only string)

Reply:
@grubberr done. 
> we usually do all types safe with nulls
so i've applied this to all types. is that correct?